From 4cf76077b51b52405c331641263f4bf4148e2266 Mon Sep 17 00:00:00 2001
From: Takashi Sakamoto <o-takashi@sakamocchi.jp>
Date: Wed, 29 Nov 2023 22:39:50 +0900
Subject: [PATCH] hwdb: ieee1394-unit-function: arrangement for Sony DVMC-DA1

A commit 6a42bdb37e39 ("hwdb: ieee1394-unit-function: add Sony
DVMC-DA1") is based on kernel feature unreleased yet (furthermore, not
merged yet). The original intension of new entry is to configure permission
of special file for FireWire character device, so this commit changes the
entry so that it can covers the issued case in existent version of Linux
kernel as out best effort.

When the new version of Linux kernel is released with the new feature,
then following commits would fulfill the hwdb with vendor and model names.

(cherry picked from commit 26c9773ab9c46db434a4dc4fca8332e1a73b706f)

Related: RHEL-5950
---
 hwdb.d/80-ieee1394-unit-function.hwdb | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/hwdb.d/80-ieee1394-unit-function.hwdb b/hwdb.d/80-ieee1394-unit-function.hwdb
index 91e8149804..75129823bb 100644
--- a/hwdb.d/80-ieee1394-unit-function.hwdb
+++ b/hwdb.d/80-ieee1394-unit-function.hwdb
@@ -1349,13 +1349,11 @@ ieee1394:ven00080046mo00000000sp0000A02Dver00010001
  IEEE1394_UNIT_FUNCTION_VIDEO=1
 
 #
-# Media converter without AV/C command set.
+# Sony Media converter without AV/C command set.
 #
 
-ieee1394:node:ven0x080046mo0xfa0000units0x00a02d:0x010000
-ieee1394:ven00080046mo00FA0000sp0000A02Dver00010000
- ID_VENDOR_FROM_DATABASE=Sony Corporation
- ID_MODEL_FROM_DATABASE=DVMC-DA1
+ieee1394:node:ven0x080046units0x00a02d:0x010000
+ieee1394:ven00080046*sp0000A02Dver00010000
  IEEE1394_UNIT_FUNCTION_VIDEO=1
 
 #
